internal/pkg/aws/acm/acm.go

```diff
@@ -42,14 +43,15 @@ func New(s *session.Session) *ACM {
 // ValidateCertAliases validates if aliases are all valid against the provided ACM certificates.
 func (a *ACM) ValidateCertAliases(aliases []string, certs []string) error {
 	validAliases := make(map[string]bool)
+	AllowedDomainsCert := make(map[string][]string)
```

Comment:
perhaps rename `AllowedDomainsCert` to something like `sansOfCert`
so that when it reads:
```go
sans := sansOfCert[cert]
```